conanfile.py

```diff
@@ -11,6 +11,7 @@ class ModuleAConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     exports_sources = "CMakeLists.txt", "src/*", "include/*"
     package_type = "shared-library"
+    languages = "C"  # Important to remove compiler.cppstd/compiler.libcxx from config
```

Comment:
Avoid the creation of different ``package_ids`` for different ``compiler.libcxx/cppstd`` settings, as if this is a C package, it is not affected by those